packages/docs/content/components/text-field/label-masked-field.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Label-Masked Field
+intro: A label-masked field is a text-field pattern that provides instant, visual feedback about how inputed data is being interpreted without creating a noisy aural experience.
+core:
+  githubLink: design-system/src/components/LabelMask
+  storybookLink: components-text-field--label-masked-currency
+---
+
+## Examples
+
+This component formats the user input in the label as the user types to provide visual feedback about the data. The text in the input field itself will not be formatted until the field loses focus, which is when we know the user has finished typing. We do this to provide a quieter and more usable experience for all users. It is an updated alternative to our [Masked Field](/components/text-field/masked-field/).
```

Comment:
I think something's missing in this explaining why it's "a quieter and more usable experience for all users." Mainly, we aren't saying why it's quieter or more inclusive.

We also aren't applying the formatting to the label.

Perhaps something like could work:
> This component mirrors user input as formatted "hint text" after the label to provide visual feedback about the data as the user is typing. The text in the input field itself will not be formatted until the field loses focus, which is when we know the user has finished typing.
>
> Because the hinting isn’t being applied to the input itself, it's less distractive and obtrusive to screen reader users, providing a quieter and more usable experience for all users. It is an updated alternative to our [Masked Field](/components/text-field/masked-field/).

Reply:
I like this 👍